### PR TITLE
[zh]Update concepts pages(part-4) for links with '/zh/' prefix, using new prefix '/zh-cn/'

### PR DESCRIPTION
**Pull request by howieyuen**
relate issue: https://github.com/kubernetes/website/issues/34501, task-6

### CODE REVIEW

File: content/zh-cn/docs/concepts/workloads/controllers/cron-jobs.md [hunk 1]
```diff
@@ -69,7 +69,11 @@ append 11 characters to the job name provided and there is a constraint that the
 maximum length of a Job name is no more than 63 characters.
 -->
 为 CronJob 资源创建清单时，请确保所提供的名称是一个合法的
+<<<<<<< HEAD
 [DNS 子域名](/zh-cn/docs/concepts/overview/working-with-objects/names#dns-subdomain-names)。
+=======
+[DNS 子域名](/zh-cn/docs/concepts/overview/working-with-objects/names#dns-subdomain-names).
+>>>>>>> fef7d4103a (Update existing pages for links with '/zh/' prefix, using new prefix '/zh-cn/')
```
[the remark this answers is not in the document]
Comment (howieyuen) [reply content/zh-cn/docs/concepts/workloads/controllers/cron-jobs.md:76]: oops, my bad

[the remark this answers is not in the document]
Comment (ydFu) [reply content/zh-cn/docs/concepts/workloads/controllers/cron-jobs.md]: Remember that the last time git pull was to the time when the PR was split this time, if [zh] is merged, a similar situation may occur.

[the remark this answers is not in the document]
Comment (ydFu) [reply content/zh-cn/docs/concepts/workloads/controllers/cron-jobs.md]: This PR was merged in this time interval,be careful.
https://github.com/kubernetes/website/pull/34484

File: content/zh-cn/docs/concepts/workloads/controllers/cron-jobs.md [hunk 2]
```diff
@@ -69,7 +69,7 @@ append 11 characters to the job name provided and there is a constraint that the
 maximum length of a Job name is no more than 63 characters.
 -->
 为 CronJob 资源创建清单时，请确保所提供的名称是一个合法的
-[DNS 子域名](/zh-cn/docs/concepts/overview/working-with-objects/names#dns-subdomain-names)。
+[DNS 子域名](/zh-cn/docs/concepts/overview/working-with-objects/names#dns-subdomain-names).
```
Comment (ydFu) [inline content/zh-cn/docs/concepts/workloads/controllers/cron-jobs.md:72]:
```suggestion
[DNS 子域名](/zh-cn/docs/concepts/overview/working-with-objects/names#dns-subdomain-names)。
```

**Comment by tengqm** [comment]
Yes. It is risky, given the number of files these PRs touch. We have to get these merged asap. Or else there will need a huge number of rebases.

**Comment by howieyuen** [comment]
> Yes. It is risky, given the number of files these PRs touch. We have to get these merged asap. Or else there will need a huge number of rebases.

got it, on my way

File: content/zh-cn/docs/concepts/workloads/controllers/cron-jobs.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (howieyuen) [reply content/zh-cn/docs/concepts/workloads/controllers/cron-jobs.md]: thanks to point out, already rebase the main branch.

**Pull request merged**